x-pack/plugins/remote_clusters/public/application/sections/components/remote_cluster_form/validators/validate_address.ts

```diff
@@ -17,14 +17,14 @@ export function isAddressValid(seedNode) {
       // no need to wait for regEx if the part is empty
       return true;
     }
-    const [match] = part.match(/[A-Za-z0-9\-]*/);
+    const [match] = part.match(/[A-Za-z0-9\-]*/)!;
```

Comment:
Are we confident that this will never return null?

Reply:
I think we should always get a match because of the `*` operator (zero or more), but your comment made me consider that the `??` will be more readable, so I updated this line.